docs/auditing.md

```diff
@@ -0,0 +1,9 @@
+# Whitehall Content Auditing
+
+Some content types in Whitehall are versioned so that we are able to explain how a content item reached its current state.
```

Comment:
Do we know which content types are versioned? Maybe it's not necessary to say but I don't know if we actually have it documented anywhere

Reply:
We can work it out I think - I think in my head I thought being slightly vague in some areas was future-proofing in case which types are auditable changes in future, but actually that's not really helping the reader!

Reply:
Hmmm, yeah you're totally right about the future proofing 🤔 

It would almost be easier to say which models don't have auditing as we're unlikely to add it, but that might be a very confusing way to put it

Reply:
Added